lix/packages/sdk/src/discussion/comment.ts

```diff
@@ -0,0 +1,36 @@
+import { Kysely } from "kysely";
+import type { LixDatabaseSchema } from "../database/schema.js";
+import { v4 } from "uuid";
+
+export async function comment(args: {
```

Comment:
Would rename to `addChangeDiscussionComment` 

- `add` to imply that something is added to the discussion
- `changeDiscussionComment` to avoid ambiguity problems down the road if we introduce other types of comments

Reply:
@samuelstroschein  addChangeDiscussionComment   >>  no developer who is not a java developer would be ok with reading and writing THAT verbose for a part of the core api. i see no reason for going that route instead of addComment or similar simple names

Reply:
@janfjohannes, I expect change discussion comments to evolve, maybe even be succeeded by a generalizable solution. Being verbose now reduces the risk of ambiguity down the road. We can always introduce an alias `addComment` later once we are sure that the concept is nailed.

Reply:
EDIT: I had a mental error, see https://github.com/opral/monorepo/pull/3121#discussion_r1757318548 

AKA yes, just name it discussion and comment. We have a generalizable solution to discuss or comment anything that happens in lix :)